coderedcms/templatetags/coderedcms_tags.py

```diff
@@ -100,7 +103,15 @@ def process_form_cell(request, cell):
         return utils.get_protected_media_link(request, cell, render_link=True)
     if utils.uri_validator(str(cell)):
         return mark_safe("<a href='{0}'>{1}</a>".format(cell, cell))
-    return localize(cell)
+
+    # This is a last resort rendering for links, used primarily for links stored in StreamForms.
+    # TODO, find a better way to handle this.
+
+    try:
+        unescaped_cell = unescape(cell)
+        return localize(mark_safe(unescaped_cell))
+    except:
+        return cell
```

Comment:
I'm not entirely sure what's going on here. Does the streamform somehow store its own links to uploaded files?

Reply:
https://github.com/coderedcorp/coderedcms/pull/171/files#diff-e8b7a6a93ddcb52b42325f9308a07806R400

I didn't have too much time to find the reasoning, but it appears that the above line is what causes the rendering of the fields.  We're extending this model with a proxy model though, so it is fixable.